docs/yaml-config.rst

```diff
@@ -154,6 +154,21 @@ the highest supported minor version will be selected.
     python:
        version: 3.5
 
+python.use_system_site_packages
+```````````````````````````````
+
+* Default: ``false``
+* Type: Boolean
+
+When true, it gives the virtual environment access to the global site-packages directory.
+Depending of the :ref:`yaml-config:build.image`,
```

Comment:
Oops! Missed typo here. `Depending of` -> `Depending on`.

Reply:
I need to take a look at some english grammar rules p:

Reply:
Please, fix this and we can merge, right?

Reply:
Ups, forgot to resolve this